### DISCUSSION TITLE
Rate limiting downloads

### DISCUSSION THREAD

**Discussion by flexyboooy** [Flexget/Flexget#3696] [Q&A]
is it possible to rate limit regular downloads, i.e. a regular file downloads not passed to, say, youtube-dl? If so, how?

Thanks
fb

**Comment by paranoidi** [comment]
Kind of, with limit_new and how often you execute a task.

**Comment by flexyboooy** [comment]
Sorry, I meant limiting as in limiting download speeds of the internal download function.

**Reply by paranoidi** [reply]
That is not supported. Use external downloader like aria or jdownloader (I think).

**Comment by flexyboooy** [comment]
Thanks. I did find a workaround that works well enough for my needs, but it is more complicated.

Would be nice if it was not necessary ;)
